ethereum/gnosis_protocol_v2/solver_names.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS gnosis_protocol_v2.solver_names
+(
+    address     bytea,
+    environment text,
+    name        text
+);
+
+\copy gnosis_protocol_v2.solver_names FROM 'https://raw.githubusercontent.com/duneanalytics/abstractions/a68ea6d2c3045d02c29c974556d0d230533a1263/ethereum/gnosis_protocol_v2/solvers.csv' with (format csv,header true, delimiter ',');
```

Comment:
TODO - Once #1263 is merged, use a permanent link to the CSV file with the solver names.

Reply:
from what i'm seeing, it's possible a URL won't work. the copy command appears to expect the file to be stored locally. while i can use a file path on my local git, i'd prefer not to put that path into our master branch. if another teammate looks to run, the path would need edited each time

Reply:
ignore that -- thanks to the legend @antonio-mendes it appears we can read from the URL as intended

Reply:
Party time!